spec.html

```diff
@@ -38,6 +38,216 @@ <h1>Well-known intrinsic objects</h1>
+	<p>An object is a <dfn id="wrapped-module-namespace-exotic-object" variants="wrapped module namespace exotic objects">wrapped module namespace exotic object</dfn> if its [[Get]] internal method use the definitions in this section, and its other essential internal methods use the definitions found in <emu-xref href="#sec-module-namespace-exotic-objects"></emu-xref>. These methods are installed by WrappedModuleNamespaceCreate.</p>
+	<p>Wrapped module namespace exotic objects have the internal slots defined in <emu-xref href="#table-internal-slots-of-wrapped-module-namespace-exotic-objects"></emu-xref>.</p>
+	<emu-table id="table-internal-slots-of-wrapped-module-namespace-exotic-objects" caption="Internal Slots of Wrapped Module Namespace Exotic Objects" oldids="table-29">
+		<table>
+			<tr>
+				<th>
+					Internal Slot
+				</th>
+				<th>
+					Type
+				</th>
+				<th>
+					Description
+				</th>
+			</tr>
+			<tr>
+				<td>
+					[[Module]]
+				</td>
+				<td>
+					a Module Record
+				</td>
+				<td>
+					The Module Record whose exports this namespace exposes.
+				</td>
+			</tr>
+			<tr>
+				<td>
+					[[Exports]]
+				</td>
+				<td>
+					a List of Strings
+				</td>
+				<td>
+					A List whose elements are the String values of the exported names exposed as own properties of this object. The list is ordered as if an Array of those String values had been sorted using %Array.prototype.sort% using *undefined* as _comparefn_.
+				</td>
+			</tr>
+			<tr>
+				<td>
+					[[Realm]]
+				</td>
+				<td>
+					a Realm Record
+				</td>
+				<td>
+					The Realm Record in which the Wrapped Module Namespace Exotic object was created.
+				</td>
+			</tr>
+		</table>
+	</emu-table>
+
+	<emu-clause id="sec-wrapped-module-namespace-exotic-objects-get-p-receiver" type="internal method">
+		<h1>
+			[[Get]] (
+				_P_: a property key,
+				_Receiver_: an ECMAScript language value,
+			): either a normal completion containing an ECMAScript language value or a throw completion
+		</h1>
+		<dl class="header">
+			<dt>for</dt>
+			<dd>a wrapped module namespace exotic object _O_</dd>
+		</dl>
+		<emu-alg>
+			1. If Type(_P_) is Symbol, then
+				1. Return ! OrdinaryGet(_O_, _P_, _Receiver_).
+			1. Let _exports_ be _O_.[[Exports]].
+			1. If _P_ is not an element of _exports_, return *undefined*.
+			1. Let _m_ be _O_.[[Module]].
+			1. Let _binding_ be ! _m_.ResolveExport(_P_).
+			1. Assert: _binding_ is a ResolvedBinding Record.
+			1. Let _targetModule_ be _binding_.[[Module]].
+			1. Assert: _targetModule_ is not *undefined*.
+			1. If _binding_.[[BindingName]] is ~namespace~, then
+				1. Return ? GetModuleNamespace(_targetModule_).
```

Comment:
This condition here depends on the answer to the question 1 from the PR description. Either way this line has to change.